specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+# Charge Subcategory
+
+Charge Subcategory indicates what kind of usage or adjustment the row represents. Charge Subcategory is a supplemental detail to the Charge Type. It provide additional context to describe the primary category of charge.
+
+This linkage to the parent ChargeType means that for every entry under ChargeType, there is a corresponding Charge Subcategory that further refines the nature of the charge. It's a nested level of detail that allows users to see not just what type of charge was incurred, but also how that charge interacts with their purchasing agreements and discounts.
+
+When Charge Type is "Usage", the Charge Subcategory specifies if the usage is accounted for by pre-purchased commitments or if the usage has not benefited from these discounts.
```

Comment:
Isn't it the role of the Pricing Model dimension to specify whether the charge is covered by a commitment or not?

Reply:
This is about whether the commitment was used or not. But I see your point about that not being clear... Thoughts on this...?
```suggestion
When Charge Type is "Usage" and the charge is covered by a commitment, the Charge Subcategory indicates whether the commitment was used or not.
```

Reply:
A usage charge may not be related to a commitment (case where on-demand rate is used). For a commitment, the charge may relate to the used or unused portion. So there are 3 cases. How are they handled?

Reply:
In that case, we're keeping this column null for now. We do think there should be a value but felt it's too late to lock it by EOW. We cover that on line 18 below.